### PR TITLE
Edits for grammar and readability. Fixed links--especially endnote links.

### PR DESCRIPTION
**Pull request by adamstallard**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -72,159 +72,159 @@ Identity Providers are responsible for validating the identity of Swift Citizens
...
+Transactions have no inherent cost on SwiftDemand, however [Identity Providers](#identity-providers) have the ability to set transaction fees--allowing them to have funds to resolve transaction disputes. The Swift Protocol reserves 80% of each block to be used exclusively by Identity Providers. The space is further partitioned by Identity Provider--proportional to the number of Swift Citizens each has verified, with a minimum requirement for inclusion of ten Swift Citizens. It's therefore the responsibility of Identity Providers to ensure that spam transactions are not added to the chain. The remaining 20% of each block will be open for transactions with a bidding system--in a similar manner to how bitcoin functions <sup>[[5]](#5)</sup>.
```
Comment (zaguios) [inline README.md:121]: The amount of transactions that an Identity Provider is able to have included on a single block can't go below 10 to allow new Identity Providers to add new Swift Citizens and make initial transactions.

File: README.md [hunk 2]
```diff
@@ -72,159 +72,159 @@ Identity Providers are responsible for validating the identity of Swift Citizens
...
 
 ### Identity Provider Salary
-The Identity Provider salary is decided on by the delegated nodes. The salary is multiplied by the amount of Swifts that have been claimed by Citizens that are under that specific Identity Provider. For example if the salary decided by the Delegated Nodes is 0.01 and the Identity Provider has 100,000 active validated citizens that claimed 5,000,000 Swifts during that day the Identity Provider will receive 50,000 Swifts at the first block after midnight UTC. 
+The _Salary Multiplier_ for Identity Providers is decided by the Delegated Nodes. An Identity Provider's salary is determined multiplying by the number of Swifts that have been claimed by Citizens under that specific Identity Provider by the Salary Multiplier. For example, if the Salary Multiplier is 0.01 and the Identity Provider has 100,000 active validated citizens that claimed 5,000,000 Swifts during that day, the Identity Provider will receive 50,000 Swifts at the first block after midnight UTC.
```
Comment (zaguios) [inline README.md]: determined by multiplying

File: README.md [hunk 3]
```diff
@@ -72,159 +72,159 @@ Identity Providers are responsible for validating the identity of Swift Citizens
...
 
 ## Reserved Funds
-20 Billion Swifts will be reserved, these Swifts are not controlled by any central source, but rather are under the control of the Delegated Nodes. It is the responsibility of the Delegated Nodes to assign these funds on an as needed basis to Identity Providers, Delegated Nodes, Swift Citizens, or Swift Accounts in an effort to further the success of the Swift Protocol.
+20 Billion Swifts will be reserved; these Swifts are not controlled by any central source, but are under the control of the Delegated Nodes. It's the responsibility of the Delegated Nodes to assign these funds on an as-needed basis to Identity Providers, Delegated Nodes, Swift Citizens, or Swift Accounts in an effort to further the success of the Swift Protocol.
```
Comment (zaguios) [inline README.md]: The but in `central source, but are under` is a bit strange. This sentence can probably just be reworded.

File: README.md [hunk 4]
```diff
@@ -47,16 +47,16 @@ The Swift Protocol is an implementation of a Decentralized Autonomous Organizati
...
 
 ### Swift Accounts
-Swift Accounts are not connected to any particularly real world identity and therefore do not receive Basic Income. These accounts still must be approved by Identity Providers before being created. An Identity Provider must be linked to all Swift Accounts, however Identity Providers are not allowed to make transactions on behalf of a Swift Account. Delegated Nodes and Identity Providers both inherit from the Swift Account type.
+Swift Accounts are not connected to any particularly real-world identity and therefore do not receive Basic Income. These accounts still must be approved by Identity Providers before being created. An Identity Provider must be linked to all Swift Accounts, however Identity Providers are not allowed to make transactions on behalf of a Swift Account. Delegated Nodes and Identity Providers both inherit from the Swift Account type.
```
Comment (zaguios) [inline README.md:59]: I have another mistake here! It should be `particular` instead of `particularly`

File: README.md [hunk 5]
```diff
@@ -72,159 +72,159 @@ Identity Providers are responsible for validating the identity of Swift Citizens
...
+When a Swift Citizen or Identity Provider requires funds to perform some task, they must submit a public funding proposal. This proposal will be written in plain text, signed with the requester's private key and then added to the blockchain. Nodes will then have one week to vote on the proposal. Non-votes are counted as "No"s. A majority consensus is required for the funds to be transferred.
 
 ## Consensus Protocol
-A new consensus mechanism is used in the Swift Protocol that allows for extremely fast block times while still being trustable. The Swift Protocol uses a modified version of Delegated Byzantine Fault Tolerance \[6] that is implemented by NEO. Instead of having each block signed by a majority of the nodes, consensus will be decided based on the longest chain.
+A new consensus mechanism is used in the Swift Protocol that allows for extremely fast block times while still being trustable. The Swift Protocol uses a modified version of the Delegated Byzantine Fault Tolerance algorithm implemented by NEO<sup>[[7]](#7)</sup>. Instead of having each block signed by a majority of the nodes, consensus will be decided based on the longest chain.
```
Comment (zaguios) [inline README.md:145]: I prefer this without `the` and `algorithm`. I also shouldn't have capitalized `delegated` in this context.

File: README.md [hunk 6]
```diff
@@ -72,159 +72,159 @@ Identity Providers are responsible for validating the identity of Swift Citizens
...
 
 ### Selecting Forgers
-When Delegated Nodes are elected, they are placed into an ordered list of blocks that are permitted to forge blocks ordered by the amount of votes received. Nodes will then take turns forging blocks round robin style. When it is a nodes turn to create a block they will have a 10 second window to forge a block. If no block is forged during this duration then the next Delegated Node on the list will also have permission to forge a new block. This will be achieved by using a timestamp server \[5].
+When Delegated Nodes are elected, they are placed into a list of blocks that are permitted to forge blocks, ordered by the amount of votes received. Nodes will then take turns forging blocks round-robin. When it's a node's turn to create a block, they will have a ten-second window to forge a block. If no block is forged during this duration, the next Delegated Node on the list will also have permission to forge a new block. This will be achieved by using a timestamp server <sup>[[6]](#6)</sup>.
```
Comment (zaguios) [inline README.md:148]: I think it's important to clarify that the list is ordered. This makes it clear how all Delegated Nodes are aware of who is allowed to currently forge a block.

File: README.md [hunk 7]
```diff
@@ -72,159 +72,159 @@ Identity Providers are responsible for validating the identity of Swift Citizens
...
 
 ## Connection To SwiftDemand
-SwiftDemand is separate from the Swift Protocol, however they are tightly connected and currently being developed by the same team. SwiftDemand will be the initial Identity Provider on the Swift Protocol and all Swifts that exist on SwiftDemand at the time of launching the Swift Mainnet will be transferred at a 1:1 ratio. Swift Citizens however will be required to undergo stricter Identity Verification requirements at this time to comply with KYC\[7] and AML\[8]. 
+SwiftDemand is separate from the Swift Protocol. They are tightly connected, however, and currently being developed by the same team. SwiftDemand will be the initial Identity Provider on the Swift Protocol and all Swifts that exist on SwiftDemand at the time of launching the Swift Mainnet will be transferred at a 1:1 ratio. Swift Citizens will be required to undergo stricter Identity Verification requirements at this time to comply with KYC<sup>[[8]](#8)</sup> and AML<sup>[[9]](#9)</sup>.
```
Comment (zaguios) [inline README.md:194]: The sentence `They are tightly connected, however, and currently` sounds a bit odd. Maybe there can be a better way to phrase this.

**Comment by zaguios** [comment]
Changes look great! I made a few comments, can we squash the commits as well?

File: README.md (hunk 1 above)
Comment (adamstallard) [reply README.md:121]: Ok, I understand what you're trying to say.  Change "amount" to "number" and "10" to "ten."

File: README.md (hunk 2 above)
Comment (adamstallard) [reply README.md]: Nice catch.

File: README.md (hunk 3 above)
Comment (adamstallard) [reply README.md]: Sounds good.  Watch the comma splice there and "but rather" is redundant.

File: README.md (hunk 4 above)
Comment (adamstallard) [reply README.md:59]: Yes I saw that, but I'm not going to touch the next few paragraphs until you fix the "Swift Accounts" naming confusion.

File: README.md (hunk 5 above)
Comment (adamstallard) [reply README.md:145]: Agreed.  Put a comma after "Tolerance" then.

File: README.md (hunk 6 above)
Comment (adamstallard) [reply README.md:148]: Me too, just avoid using the word "ordered" twice, if possible.

File: README.md (hunk 7 above)
Comment (adamstallard) [reply README.md:194]: Probably--or instead of changing the word order, change the comma to a semi-colon so it's not a comma splice.

**Comment by adamstallard** [comment]
Yes, squash and merge on your end, and then put in the fixes you put in your comments.

**Pull request merged**